components/test_raftstore/Cargo.toml

```diff
@@ -18,3 +18,8 @@ rand = "0.3"
 slog = { version = "2.3", features = ["max_level_trace", "release_max_level_debug"] }
 # better to not use slog-global, but pass in the logger
 slog-global = { version = "0.1", git = "https://github.com/breeswish/slog-global.git", rev = "91904ade" }
+
+[dependencies.engine_rocksdb]
+git = "https://github.com/pingcap/rust-rocksdb.git"
+rev = "5f49f9ef202d8ad9515de2bb05c8395c1c66b8aa"
+package = "rocksdb"
```

Comment:
This is unfortunate, and I think unnecessary. After the previous PR all the rocksdb imports used project-wide should have been reexported through a single module in tikv, and after this pr the abstraction boundary should is `tikv::engine`. If this crate then depends concretely on rocksdb then it's violating an abstraction layer and will probably need to be removed again later. If you agree will you try to remove this and import everything needed from tikv?

Reply:
You are right, I will remove it.